components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/device/dao/SQLQueries.java

```diff
@@ -46,7 +46,7 @@ private SQLQueries() {
 
         public static final String GET_CONSUMER_KEY_FOR_DEVICE_CODE = "SELECT CONSUMER_KEY FROM (SELECT * " +
                 "FROM IDN_OAUTH_CONSUMER_APPS INNER JOIN IDN_OAUTH2_DEVICE_FLOW ON IDN_OAUTH_CONSUMER_APPS.ID = " +
-                "IDN_OAUTH2_DEVICE_FLOW.CONSUMER_KEY_ID) WHERE DEVICE_CODE = ?";
+                "IDN_OAUTH2_DEVICE_FLOW.CONSUMER_KEY_ID) AS CONSUMER_APPS_WITH_DEVICE_FLOW WHERE DEVICE_CODE = ?";
```

Comment:
`AS` keyword is not supported in some places in Oracle DB. This seems to be such a place.

Reply:
Yes, we don't want to put AS keyword in front of derived table CONSUMER_APPS_WITH_DEVICE_FLOW. AS keyword is not a mandatory keyword in SQL queries. it's an optional we can skip because DBs like Oracles doesn't support AS keyword. 

`                "IDN_OAUTH2_DEVICE_FLOW.CONSUMER_KEY_ID) CONSUMER_APPS_WITH_DEVICE_FLOW WHERE DEVICE_CODE = ?";
this will be fine
`

Reply:
Yes, we don't want to put AS keyword in front of derived table CONSUMER_APPS_WITH_DEVICE_FLOW. AS keyword is not a mandatory keyword in SQL queries. it's an optional we can skip because DBs like Oracles doesn't support AS keyword. 

`                "IDN_OAUTH2_DEVICE_FLOW.CONSUMER_KEY_ID) CONSUMER_APPS_WITH_DEVICE_FLOW WHERE DEVICE_CODE = ?";
this will be fine
`
you can [refer to ](https://github.com/wso2-extensions/identity-inbound-auth-oauth/blob/2e1e32e70e39d64b3af934f730dcc7d9f2b3c8c0/components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/openidconnect/dao/SQLQueries.java#L97)  GET_IDN_OIDC_SCOPES_CLAIMS query..

Reply:
@madurangasiriwardena @sarubi Thank guys! You probably saved me from an additional testing round. Please find the needful fix in 5596967e09377e503dad4f94e0b674f221be4a85.